library/lua/argparse.lua

```diff
@@ -4,6 +4,10 @@ local _ENV = mkmodule('argparse')
 
 local getopt = require('3rdparty.alt_getopt')
 
+---@nodiscard
+---@param args string[] Most commonly `{ ... }`
+---@param validArgs table<string, integer> Use `utils.invert`
```

Comment:
the value of this table just has to be truthy. maybe use boolean instead of integer

Reply:
If we do that the return value of `utils.invert` will be incompatible in LuaLS's eyes. We could do "any" and mention in the comment it has to be truthy?

Just to be clear:

```lua
---@param truthy boolean
function foo(truthy) end

local bar = foo("") -- Shows a warning
local caz = foo(1) -- Shows a warning
```